data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+	description "Travel to the <waypoints> system to fight and disable a stranded Ra'gru Ik 618. Do not destroy or capture the ship. Our intel says the world ship has a large civilian population."
+	source
+		government "Remnant"
+		not attributes "station"
+	waypoint "Parca"
+	to offer
+		has "event: remnant: cognizance calm"
+		"combat rating" > 8103
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Ra'gru Ik 618 to be precise. Normally we would request you track down this ship and eliminate, but our scouts also reported that this appears to be home to a large civilian population."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arranged for a prisoner swap with the Korath."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have intention of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Lets make sure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment. "I feared my ask was too much. I more than understand the sentiment. Maybe I was being naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Parca"
+		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"
+	on enter "Parca"
+		dialog
+			`As you enter the sector your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of the children.`
+	on complete
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`
+			`	"She explained that on her last cloaked fly by scanning run of the 618 she noticed what looked like large group of Korath children playing some type of ball game on the thermal scan."`
+			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 uniq bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
+			`	"I figured you where those children's best chance. I am glad I was not wrong." he says with a content smile on his face.`
+			`	"I worry in our ignorance of Korath society we may have unnecessarily destroyed other ships like this one..."`
+			`	"Expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```

Comment:
```suggestion
			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these bounties with the proper restraint, expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```

Reply:
Updated with your suggested text, but couldn't use the commit button because of other changes I had already accepted.